storageAPI/storage/api.py

```diff
@@ -1,109 +0,0 @@
-import uuid
-
```

Comment:
Remember, you will need the storage/api.py of COMPSs and the StorageItf java code to make Hecuba work with COMPSs